README.md

```diff
@@ -434,6 +435,9 @@ The threadpool is enabled by default, and starts with 5 threads that it can incr
 
 To disable threadpool, run `cmake .. -DENABLE_KVS_THREADPOOL=OFF`
 
+### Thread stack sizes
+The default thread stack size for the KVS WebRTC SDK is 64 kb. Notable stack sizes that may need to be changed for your specific application will be the ConnectionListener Receiver thread and the media sender threads. Please modify the stack sizes for these media dependent threads to be suitable for the media your application is processing.
```

Comment:
`ConnectionListener Receiver thread` : I see it being hard coded. How would the customer modify this? 

Also, can we add an example of how to modify this? Basically the usage example in the media sender APIs to add more clarity?

Reply:
Are you proposing a compile time or run time modification?

Reply:
I am ok if it is hard coded and we dont have any option, but if we do not want the customer to modify this, we should just remove the text about ConnectionListener from the README. Ideally, when seeing this, customers would not want to dig into the file to hard code and create a patch to support their use case - it is a hassle.